docs/tutorial/represented-file.md

```diff
@@ -20,23 +20,12 @@ To set the represented file of window, you can use the
 
 ## Example
 
-Starting with a working application from the
-[Quick Start Guide](quick-start.md), add the following lines to the
-`main.js` file:
-
 ```javascript fiddle='docs/fiddles/features/represented-file'
-const { app, BrowserWindow } = require('electron')
-
-app.whenReady().then(() => {
-  const win = new BrowserWindow()
-
-  win.setRepresentedFilename('/etc/passwd')
-  win.setDocumentEdited(true)
-})
 ```
```

Comment:
Since the Fiddle embed replacement doesn't render in the old website, we should still keep the contents here, I think, since that's all you need to make the demo work.

We might also want to explain what the APIs we're using are.